views/agency-requests/agency_requests.php

```diff
@@ -16,13 +16,41 @@
             <br />
             <form class="search-agency" method="POST" action="/agency-requests">
                 <div class="form-group">
-                    <input type="text" class="form-control" name="searchquery" placeholder="Begin typing participant's name...">
-                </div>
-                <button type="submit" class="btn cpca form-control">Submit</button>
+                    <input type="text" class="form-control user-search" name="searchquery" placeholder="Begin typing participant's name...">
+                <ul class='list-group'>
+				</ul>
```

Comment:
When the list does drop down, I would suggest adding a hover effect (cursor -> `pointer`, background -> darker), so the user knows they can click on it.